app/questionnaire/placeholder_parser.py

```diff
@@ -0,0 +1,64 @@
+from app.data_model.answer_store import AnswerStore
+from app.questionnaire.placeholder_transforms import PlaceholderTransforms
+
+
+class PlaceholderParser:
+    """
+    Parses placeholder statements from a schema dict and returns a map of their
+    final values
+    """
+    def __init__(self, answer_store=None, metadata=None, lists=None):
+        self._lists = lists
+        self._answer_store = AnswerStore() if answer_store is None else answer_store
```

Comment:
Maybe?
```python
self._answer_store = answer_store or AnswerStore()
```

Reply:
Maybe what?

Reply:
There was a code quote in Mebin's comment

Reply:
Yeah, I saw it. I was being slightly facetious.